packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/map_configuration.dart

```diff
@@ -152,6 +153,9 @@ class MapConfiguration {
   @Deprecated('cloudMapId is deprecated. Use mapId instead.')
   String? get cloudMapId => mapId;
 
+  /// Preferred color scheme for the map in web platform
```

Comment:
Please follow the style of the other platform-specific field comments in this class.